DEV-README.md

```diff
@@ -0,0 +1,74 @@
+# Dev Compose Setup Requirements
+There are 3 different sets of metric logs being sent to logstash's elasticsearch:
+1. Apache HTTP logs => Logstash => elasticsearch
+2. Dropwizard Metrics => Metricbeats => elasticsearch
```

Comment:
Might want to experiment with https://www.elastic.co/blog/using-beats-with-elasticsearch-on-aws in November